docs/introduction/faq.md

```diff
@@ -0,0 +1,121 @@
+[Is Talawa free?](#q-is-talawa-free)  
+[How do I use Talawa?](#q-how-do-i-use-talawa)  
+[How do I create account on Talawa?](#q-how-do-i-create-account-on-talawa)  
+[How do I list my organization on Talawa?](#q-how-do-i-list-my-organization-on-talawa)  
+[How can my organization benefit from Talawa?](#q-how-can-my-organization-benefit-from-talawa)
+
+---
+
+### What is the Talawa Project?
+Talawa project is open source software for managing Community Based Organizations
+
+[↑ table of contents](#table-of-contents)
+
+---
+
+### Why should I use Talawa?
+Talawa provides help to community based organizations collaborate with their membership. These organizations would include religious groups, non-profit charities, social groups and in limited cases, businesses.​
+
+[↑ table of contents](#table-of-contents)
+
+---
+
+### What does 'Talawa' mean, how do I pronounce it?
+Talawa is a Jamaican Patois word that means to be strong, fearless, strong-willed, and not to be underestimated. Jamaicans have always had a motto that encapsulates this idea of being great: 'We likkle but we tallawah' It means that we may be small, but we are strong, we are mighty, and can do anything. It is pronounced exactly as it is spelled: /Tala-wa/.
+
+[↑ table of contents](#table-of-contents)
+
+---
+
+### What services are provided by Talawa?
+Talawa has three main components:
+1. A mobile application with social media features
+2. A web based portal to be used by the organization’s administrative team, and
+3. An API providing access to data and features.​​
+
+[↑ table of contents](#table-of-contents)
+
+---
+
+### How is Talawa different from other community based organizations?
+Though software applications exist for these types of organizations (eg. Breeze ChMS, The Realm), they assume their memberships are fully literate and have access to email. But we wanted to create a system that would work for countries with similar technology and education challenges as Jamaica.​
+
+[↑ table of contents](#table-of-contents)
+
+---
+
+### Is Talawa safe?
+Talawa is safe, uses various encryption technologies to protect sensitive data. One of the ideas is to add SSL. SSL (Secure Socket Layer) helps to create a secure encrypted connection between the server (our project) and the client and protects Talawa from hacking and other security threats.
+
+[↑ table of contents](#table-of-contents)
+
+---
+
+### What can a leader of a non-profit do with Talawa?
+A leader of a non-profit can set up their organization within Talawa. This will allow them to do things like plan events, coordinate volunteers, and post updates.
+
+[↑ table of contents](#table-of-contents)
+
+---
+
+### What is Palisadoes?
+The Palisadoes is a Foundation that aims to provide new avenues of promoting Jamaica’s technology talent through a series of programs which create increased awareness of the island’s abilities in the global marketplace. It was founded by Jamaicans living in the United States.​
```

Comment:
Please change to:  
The Palisadoes Foundation is a 501(c)(3) non-profit organization. Our mission is to provide equitable access to life opportunities for STEM education recipients. 

Palisadoes was created by a group of expatriate Jamaican information and communication technology (ICT) professionals. They wanted to assist the continued development of new and existing technologies for the benefit of the island. 

The Foundation quickly realized that there was universal need for these services in underserved communities where the Jamaican diaspora live. We have expanded our service region to include them.